docs/release-notes/cloud/cloud-2023-11.md

```diff
@@ -0,0 +1,84 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud November 2023.
+included_jira_versions: ['2023.11']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.80
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.123
+---
+
+# Cloud November 2023
+
+These release notes are for the Codacy Cloud updates during November 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added [support for Semgrep’s open-source engine](../../getting-started/supported-languages-and-tools.md) to perform static application security testing (SAST), with the option to upgrade to Semgrep Pro. (TAROT-2347)
+-   You now get two additional coverage status reports from the new, faster Coverage engine on your Git provider, showing whether your diff coverage and coverage variation are up to standard or not as configured on the quality gate rules for your repository. (ALA-624)
+
+    !!! note
+        This setting is active by default. [If you're using GitHub, see how to update your setup to use the new Coverage status reports](./cloud-2023-11-23-new-coverage-engine-status-checks.md).
+
+## Bug fixes
+
+-   The Codacy Coverage engine can now process coverage reports that contain an arbitrarily large number of hits for each covered line. (ALA-660)
+
+## Deprecations
+
+-   Dropped support for repository-level Slack, Jira, and Webhook integrations. For more information, [see the deprecation notice](./cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md). (CY-7202, CY-7203, CY-7204)
```

Comment:
I would use "feature removal" instead of "deprecation" because:

1. (From a quick Google search,) "Deprecation" implies that the deprecated feature continues to function as normal until removal, which is not the case, as we already removed the UI.
2. "Removal" is more aligned with the deprecation notice.
3. Also, it's not a breaking change, as existing integrations will keep working.

I would also add that existing integrations will keep working, as it's short enough to fit the release note, and might be relevant for users using the feature.

WDYT?

```suggestion
## Feature removal

-   We removed the option to create, configure, and delete repository-level Slack, Jira, and Webhook integrations. Your existing integrations will keep working, but you can no longer configure or delete them on the Codacy app. For more information, [see the deprecation notice](./cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md). (CY-7202, CY-7203, CY-7204)
```

Reply:
LGTM 👍